### ISSUE TITLE
Action on submit - Redirect to an entry - Could not get this value using graphql

### ISSUE THREAD

**Issue by vignesh511989** [verbb/formie#501]
**Description**
I'm using GraphQL to get the redirect entry from the form setting. But it does not have any field to retrieve the attached entry url.

**Steps to reproduce**
1. Craft cms headless with formie. Create a new form and in the behaviour tab choose "Redirect to an entry". 
2. Use Craft Graphiql to get the redirect entry.

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: Yes or No

**Additional info**
- Plugin version: 1.3.20
- Craft version: 3.6.14
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.7",
  "...": "..."
}
```

Then run `composer update`.

You'll be able to use:
```
{
  form(handle:"gql") {
    settings {
      submitMethod
      submitAction
      submitActionTab
      submitActionUrl
      redirectUrl
      redirectEntry {
        url
      }
    }
  }
}
```

```
{
  "data": {
    "form": {
      "settings": {
        "submitMethod": "ajax",
        "submitAction": "entry",
        "submitActionTab": "same-tab",
        "submitActionUrl": "",
        "redirectUrl": "http://formie.com/my-entry",
        "redirectEntry": {
          "url": "http://formie.com/my-entry"
        }
      }
    }
  }
}
```

**Comment by engram-design** [comment]
Fixed in [1.4.8](https://github.com/verbb/formie/releases/tag/1.4.8)
